cartoframes/viz/widget.py

```diff
@@ -20,50 +20,50 @@ class Widget(object):
 
         from cartoframes.viz import Widget
 
-        Widget({
-            type: 'formula',
-            value: 'viewportSum($amount)'
-            title: '...',
-            description: '...',
-            footer: '...'
-        })
+        Widget('formula', value='viewportSum($amount)', title='...')
     """
 
-    def __init__(self, data=None):
-        self._init_widget(data)
+    def __init__(self, f_arg, **kwargs):
+        if isinstance(f_arg, dict):
+            self._init_widget(f_arg)
+        elif isinstance(f_arg, str):
+            self._init_widget(kwargs, f_arg)
+        else:
+            raise ValueError('Wrong widget input.')
 
-    def _init_widget(self, data):
+    def _init_widget(self, data, widget_type=''):
         self._type = ''
         self._value = ''
-        self._name = ''
         self._title = ''
+        self._prop = ''
         self._description = ''
         self._footer = ''
         self._options = {}
+
         if data is not None:
-            if isinstance(data, dict):
-                self._type = data.get('type', '')
-                self._value = data.get('value', '')
-                self._name = gen_variable_name(self._value)
-                self._title = data.get('title', '')
-                self._description = data.get('description', '')
-                self._footer = data.get('footer', '')
-                self._options = camel_dictionary(data.get('options', {}))
-            else:
-                raise ValueError('Wrong widget input.')
+            self._type = widget_type if widget_type else data.get('type', '')
+            self._value = data.get('value', '')
+            self._title = data.get('title', '')
+            self._description = data.get('description', '')
+            self._footer = data.get('footer', '')
+            self._options = camel_dictionary(data.get('options', {}))
+            self._prop = data.get('prop', '')
```

Comment:
I would add here explicitly that the default prop is `filter`

Reply:
Fixed, added only when is an animation widget for now